ngraph/frontend/onnx_import/src/op/org.openvinotoolkit/prior_box.cpp

```diff
@@ -83,6 +84,48 @@ namespace ngraph
                         axes)};
                 }
 
+                OutputVector prior_box_clustered(const Node& node)
+                {
+                    auto inputs = node.get_ng_inputs();
+                    NGRAPH_CHECK(inputs.size() == 2, "Invalid number of inputs");
+
+                    auto output_shape_rank = inputs[0].get_partial_shape().rank().get_length();
+                    auto image_shape_rank = inputs[1].get_partial_shape().rank().get_length();
+                    CHECK_VALID_NODE(node,
+                                     output_shape_rank == 4,
+                                     "Only 4D inputs are supported. First input rank: ",
+                                     output_shape_rank,
+                                     " (should be 4)");
+                    CHECK_VALID_NODE(node,
+                                     image_shape_rank == 4,
+                                     "Only 4D inputs are supported. Second input rank: ",
+                                     image_shape_rank,
+                                     " (should be 4)");
+
+                    auto output_shape = std::make_shared<default_opset::ShapeOf>(inputs[0]);
+                    auto image_shape = std::make_shared<default_opset::ShapeOf>(inputs[1]);
+                    auto output_shape_slice = detail::make_slice(output_shape, 2, 4);
+                    auto image_shape_slice = detail::make_slice(image_shape, 2, 4);
+
+                    ngraph::op::PriorBoxClusteredAttrs attrs{};
+                    attrs.widths = node.get_attribute_value<std::vector<float>>("width");
+                    attrs.heights = node.get_attribute_value<std::vector<float>>("height");
+                    attrs.clip = static_cast<bool>(node.get_attribute_value<int64_t>("clip", 0));
+                    attrs.variances =
+                        node.get_attribute_value<std::vector<float>>("variance", {0.1f});
+                    attrs.step_heights = node.get_attribute_value<float>("step_h", 0.0f);
+                    attrs.step_widths = node.get_attribute_value<float>("step_w", 0.0f);
+                    attrs.offset = node.get_attribute_value<float>("offset", 0.0f);
+
+                    auto axes = default_opset::Constant::create(
+                        element::i64, Shape{1}, std::vector<int64_t>{0});
+
+                    return {std::make_shared<default_opset::Unsqueeze>(
```

Comment:
Why is this `Unsqueeze` needed here?

Reply:
I have added this Unsqueeze to match pattern from: **ConvertPriorBoxClusteredToLegacy** transformation.
I'm not sure what is the purpose of the Unsqueeze there, @itikhono, could you please help us?

Reply:
The semantics of the custom PriorBox (Clustered) operator in ONNX and the specified version of the op in OV are different.
We use this sub-graph to convert ONNX PriorBox to the spec version (Unsqueeze is required)
![image](https://user-images.githubusercontent.com/21181928/107806401-dfe82180-6d77-11eb-94bd-39850ae188de.png)
But our plugins support only ONNX version, so we try to detect exactly this pattern and convert it from our specified version to supported by plugins using ConvertPriorBoxClusteredToLegacy.
Note: This code should be the same as PriorboxMutation transformation in MO. Please double-check it.